libs/designsystem/src/lib/components/progress-circle/progress-circle.component.html

```diff
@@ -1,5 +1,10 @@
 <div class="transcluded-content">
   <ng-content></ng-content>
 </div>
-<kirby-progress-circle-ring [themeColor]="themeColor" [value]="shownValue" [radius]="radius">
+<kirby-progress-circle-ring
+  [themeColor]="themeColor"
+  [value]="shownValue"
+  [radius]="radius"
+  [size]="size"
```

Comment:
To my eyes it looks a bit odd to have both a `size` and a `radius` input property. When I read it, I kinda expect the size to also control the radius. 

Could it be changed to:  `stroke-size`, `stroke-width`, `stroke` or similar?

Reply:
Could a solution be to move the `STROKE_MAP` to `progress-circle-component` such that the progress-circle is responsible for setting the desired `stroke-width` in px? Then `strokeWidth` can be an input-property taking an integer 😄

Reply:
Good suggestion, as we talked about on slack, I will create a config object on  `progress-circle.component.ts`, and calculate the stroke with in the component, so we avoid a size input on the progress-ring component. 

This way we can extend the config with other attributes later.